subiquity/server/controllers/kernel.py

```diff
@@ -62,6 +62,7 @@ def start(self):
                 with open(mp_file) as fp:
                     kernel_package = fp.read().strip()
                 self.model.metapkg_name = kernel_package
+                self.model.explicitly_requested = True
```

Comment:
A fun scenario here is that if we trigger this one, it's (usually) an image construction mismatch, not (usually) autoinstall.  It could be autoinstall with early-commands. 

I think the right answer is that we don't want to set `explicitly_requested = True` on this code path.

Note that https://git.launchpad.net/casper/tree/scripts/casper-bottom/61desktop_canary_tweaks#n36 selects the HWE kernel by way of the /run/kernel-meta-package today, and presumably /etc/subiquity/kernel-meta-package on the long term.

Reply:
Good point, thanks!